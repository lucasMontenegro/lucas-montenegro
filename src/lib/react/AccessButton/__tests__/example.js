describe(`../example`, () => {
  jestUtils.describeDependencies({
    deps: [
      `react`,
      `lib/react/auth0`,
      `@material-ui/core`,
      `../index.js`,
      `lib/languageDetector`,
      `lib/react/DarkMode`,
      `lib/react/Theme`,
      `lib/react/CssBaseline`,
      `react-router-dom`,
    ],
    relativeBasePath: __dirname,
  })
})