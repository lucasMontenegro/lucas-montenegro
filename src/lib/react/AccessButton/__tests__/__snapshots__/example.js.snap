// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`../example external dependencies @material-ui/core should match the installed version 1`] = `"^4.5.0"`;

exports[`../example external dependencies react should match the installed version 1`] = `"^16.10.1"`;

exports[`../example external dependencies react-router-dom should match the installed version 1`] = `"^5.1.2"`;

exports[`../example local dependencies ../index.js should match the saved hash 1`] = `"da3967f39928f2ecc58fd5e85909ebae3339e4916fcc1b99f9e3ea02b1f44567"`;

exports[`../example local dependencies lib/languageDetector should match the saved hash 1`] = `"f9c2fb9d79c6b6722c4bebf69fd3b6ae89c04240901b2298a9711e1007b4eace"`;

exports[`../example local dependencies lib/react/CssBaseline should match the saved hash 1`] = `"11eed0723df2013870f1dd74eda1682d2ec8ecfb27058ed7d9b014b096f8b582"`;

exports[`../example local dependencies lib/react/DarkMode should match the saved hash 1`] = `"b884ab2b8b07e08294620261dd8e79eba68dc11d894c05810e8465c30feb1231"`;

exports[`../example local dependencies lib/react/Theme should match the saved hash 1`] = `"45fa27ae3358adf786581e4b490d426bdcbb57b3f0bea831d5d7d5ad55378565"`;

exports[`../example local dependencies lib/react/auth0 should match the saved hash 1`] = `"ee43a5ff11bb38d21d915d88c963ea0440ffb8f5d5b3d9bfcb0c297cc3a53448"`;
